app/Http/Controllers/AfterPaddleBundleSaleController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Bundle;
+use App\Models\Referrer;
+use Illuminate\Http\Request;
+
+class AfterPaddleBundleSaleController
+{
+    public function __invoke(Request $request, Bundle $bundle)
+    {
+        sleep(4);
+
+        Referrer::forgetActive();
+
+        session()->flash('sold_purchasable', $bundle->purchasables->first());
+
+        if (current_user()) {
+            session()->flash('latest_purchase', current_user()->purchases()->latest()->first());
+        }
+
+        flash()->success('Purchase successful!');
+
+        return redirect()->route('products.show', $bundle->purchasables->first()->product);
```

Comment:
We redirect to the first product inside the bundle after the sale here. As bundles don't really have a dedicated detail page right now (I opted to skip this and go straight to the "buy" page).

We could redirect to the user's purchases instead?

Reply:
Redirecting to the user's purchases seems like a good solution. A flash message should indicate that new products have been added.